format/index.md

```diff
@@ -234,7 +232,9 @@ A link object that represents a to-many relationship **MAY** also contain
 pagination links, as described below.
 
 If a link object refers to resource objects included in the same compound
-document, it **MUST** include object linkage to those resource objects.
+document, it **MUST** include resource linkage to those resource objects.
+This allows a client to link together all of the included resource objects
+without having to `GET` one of the relationship URLs.
```

Comment:
I consider this last sentence non-normative and think it would be better as a note, perhaps with even further explanation.